src/Groups/directproducts.jl

```diff
@@ -361,15 +361,17 @@ julia> C = cyclic_group(2)
 Pc group of order 2
 
 julia> A = automorphism_group(Q)
-Aut( <pc group of size 8 with 3 generators> )
+Automorphism group of
+  pc group of order 8
 
 julia> au = A(hom(Q,Q,[Q[1],Q[2]],[Q[1]^3,Q[2]^3]))
 [ x, y ] -> [ x*y2, y*y2 ]
 
 julia> f = hom(C,A,[C[1]],[au])
 Group homomorphism
   from pc group of order 2
-  to aut( <pc group of size 8 with 3 generators> )
+  to automorphism group of
+    pc group of order 8
```

Comment:
Hmmm... probably should distinguish between "one line" and "detailed" printing mode here... perhaps it should be
```suggestion
  to automorphism group of pc group of order 8
```
or in fact just
```suggestion
  to Aut( pc group of order 8 )
```
or so ... ?

Reply:
At first sight, `Aut( ... )` looks too "untextual", compared to many other print formats.
If `Aut( ... )` is o.k. for the one line printing then why is it not good for the detailed printing?
(For example, `Sym(4)` is used in both situations.)

Reply:
Sym(4) is on its way out as well, but in a separate PR (see PR #2878 from where all this is coming)

Reply:
So some options here would include
```suggestion
  to automorphism group of pc group
```
or perhaps really this should be using that the group has been assigned to a variable `C` ...
```suggestion
  to automorphism group of C
```
or
```suggestion
  to Aut( C )
```
or for that matter, since the automorphism group has been assigned to variable `A`, it could be
```suggestion
  to A
```

Reply:
while I like the last one (`to A`), this is different from all other morphism types in Oscar. They usually print the oneline printing of domain and codomain, but the `@show_name` only triggers for terse printing.

IMO, this should be either `to automorphism group of pc group` or `to automorphism group of C`

Reply:
I vote against `Aut( ... )` output, in particular because eventually we have to fix the output:
Currently also subgroups of the full automorphism group of a group are printed as `Aut( ... )`.
My proposal for a fix is to distinguish between `automorphism group of ...` and `group of automorphisms of ...`.
(This fix should be done in a separate pull request.)